packages/linear-progress/README.md

```diff
@@ -1,64 +1,134 @@
-# mwc-linear-progress
+# `<mwc-linear-progress>` [![Published on npm](https://img.shields.io/npm/v/@material/mwc-linear-progress.svg)](https://www.npmjs.com/package/@material/mwc-linear-progress)
 
-> :warning: These components are a work in progress. They are pre-release and should be considered experimental, as they may undergo major changes before release. We are experimenting with alternate architectures and approaches with the goal of allowing us to bring the most correct and optimal implementation of Material components to the widest possible audiences. Visible progress may be slow, as this research is across teams and repositories so is not consistently reflected in commits to this codebase. :warning:
+> IMPORTANT: The Material Web Components are a work in progress and subject to
+> major changes until 1.0 release.
 
-A [Material Components](https://material.io/components/) linear-progress implementation using [Web Components](https://www.webcomponents.org/introduction)
+Progress indicators express an unspecified wait time or display the length of a process.
```

Comment:
Why is this reversed from the MDC description? Prefer "Progress indicators display the length of a process or express an unspecified wait time."

Reply:
lifted directly from here:
https://material.io/components/progress-indicators/#